terraform/aws/infrastructure.tf

```diff
@@ -49,6 +49,10 @@ resource "aws_internet_gateway" "igw" {
   count  = var.vpc_id == "" ? 1 : 0
   vpc_id = local.vpc_id
 
+  timeouts {
+    delete = "70m"
```

Comment:
You add an AWS global destroy_timeout variable in AWS variables.tf, propagate it down to the HANA module, and use both in instances and IGW, then in IGW you `+10`